### PR TITLE
Update program/README

### PR DESCRIPTION
**Pull request by ghost**
* add titles
* modify description about classes

### CODE REVIEW

File: program/README.md [hunk 1]
```diff
@@ -49,13 +49,13 @@
...
----
+## 纠错与更新
 
-培养方案时有调整和更新，若与实际不符，请联系我们
+培养方案时有调整和更新，我们会及时跟进。若您发现本站发布的内容与实际不符，请联系我们！
 
-**联系方法**
+### 联系方法
 
 1. QQ上私戳钱小辅
```
Comment (xjtu-blacksmith) [inline program/README.md]: Add contact info here

**Review by xjtu-blacksmith — CHANGES_REQUESTED** [review]
Add contact info

File: program/README.md (hunk 1 above)
Comment (ghost) [reply program/README.md]: Okay.

**Review by xjtu-blacksmith — APPROVED** [review]
Perfect! :+1:

**Pull request merged**